db/migrate/20170321012632_fill_in_content_view_components.rb

```diff
@@ -0,0 +1,32 @@
+class FillInContentViewComponents < ActiveRecord::Migration
+  class FakeContentView < ActiveRecord::Base
+    self.table_name = 'katello_content_views'
+
+    has_many :content_view_components, :class_name => "Katello::FakeContentViewVersion", :dependent => :destroy,
```

Comment:
shouldnt this be something like FillInContentViewComponents::FakeContentViewVersion ?